src/createServer.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+import express from 'express';
+import { authRouter } from './routers/auth.router.js';
+import { usersRouter } from './routers/users.router.js';
+import cors from 'cors';
+import { errorMiddleware } from './middlewares/error.middleware.js';
+import cookieParser from 'cookie-parser';
+
+export const createServer = () => {
+  const app = express();
+
+  const corsOptions = {
+    origin: '*',
```

Comment:
Using '*' for the 'origin' in CORS options allows requests from any domain, which might not be secure for production environments. Consider specifying allowed origins or using environment variables to control this setting.